mutters-ink-bot/src/main/java/com/rabidgremlin/mutters/bot/ink/InkBot.java

```diff
@@ -141,7 +142,7 @@ public InkBot(T configuration)
     List<String> defaultResponses = configuration.getDefaultResponses();
     if (defaultResponses != null)
     {
-      setDefaultResponses(defaultResponses.toArray(new String[defaultResponses.size()]));
+      setDefaultResponses(defaultResponses.toArray(new String[0]));

```

Comment:
Why did you change this ? I believe it causes a double allocation of the array ?

Reply:
As of Java8 this way of allocating arrays is the most efficient.